packages/opencensus-web-all/src/export-initial-load.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {adjustPerfTimeOrigin, tracing} from '@opencensus/web-core';
+import {OCAgentExporter} from '@opencensus/web-exporter-ocagent';
+import {clearPerfEntries, getInitialLoadRootSpan, getPerfEntries} from '@opencensus/web-instrumentation-perf';
+
+import {OpenCensusWebConfig, WindowWithOcwGlobals} from './types';
+
+const windowWithOcwGlobals = window as WindowWithOcwGlobals;
+
+/** How long to wait after `load` event to export initial load spans. */
+const WAIT_TIME_AFTER_LOAD_MS = 2000;
+
+/** Trace endpoint in the OC agent. */
+const TRACE_ENDPOINT = '/v1/trace';
+
+/**
+ * Waits until after the document `load` event fires, and then uses the
+ * `window.ocwConfig` settings to configure an OpenCensus agent exporter and
+ * export the spans for the initial page load.
+ */
+export function exportRootSpanAfterLoadEvent() {
+  const config = windowWithOcwGlobals.ocwConfig;
+  if (!config) return;
+  if (!config.agent || !config.sampled) return;
```

Comment:
Should we log a warning or a debug message here?

Reply:
Sure, I added a log message here.